app/lib/time-since/index.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+/**
+ * Shows how much time has passed since now and a given timestamp
+ */
+import _ from 'lodash';
+
+const timeSince = (timestamp: number): string => {
```

Comment:
Dates and times are pretty hard and pretty easy to get wrong. Isn't there a library we can use to pretty print this? Especially since this function isn't internationalized.